Gems/AtomLyIntegration/AtomViewportDisplayIcons/Code/Source/AtomViewportDisplayIconsSystemComponent.cpp

```diff
@@ -146,14 +146,14 @@ namespace AZ::Render
             return;
         }
 
+        const AZ::Vector2 viewportSize = [viewportContext] {
+            const auto [width, height] = viewportContext->GetViewportSize();
+            return AzFramework::Vector2FromScreenSize(AzFramework::ScreenSize(width, height));
+        }();
```

Comment:
You could potentially save `AzFramework::ScreenSize(width, height)` as a variable named `screenSize` for use here and later on line 183

Reply:
That's a good spot, I wasn't sure of the best way to do this when refactoring it hence the slight duplication, I'll update this now 👍

Reply:
This is now fixed 👍